layers/+tools/sailfish-scratchbox/packages.el

```diff
@@ -0,0 +1,39 @@
+;;; packages.el --- sailfish scratchbox layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2017 Victor Polevoy
+;;
+;; Author: Victor Polevoy <fx@thefx.co>
+;; URL: https://github.com/vityafx/sailfish-scratchbox-layer
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: MIT
+;;; Commentary:
+;;
+;; Commands
```

Comment:
The entire "Commands" section is unnecessary - we've got README.org for that, and this file isn't a package. Please remove.